test/asserting/__init__.py

```diff
@@ -0,0 +1,5 @@
+try:
+    # 3.0 and later
+    from itertools import zip_longest
+except ImportError:
+    from itertools import izip_longest as zip_longest
```

Comment:
I feel this file name is not good. 
Because the function `zip_longest` is not a asserting function.

I think it is better that move to a vint-wide namespace such as `vint.compat.itertools`.